index.js

```diff
@@ -3,31 +3,48 @@ const bookAuthor = document.getElementById('bookAuthor');
 const bookForm = document.getElementById('bookForm');
 const bookRec = document.getElementById('books-record');
 
+const booksrecord = JSON.parse(localStorage.getItem('bookArchive')) || [];
```

Comment:
- Following the project requirements, you should use classes instead of arrays and objects. Let's create a `Books` class with class methods to add and remove books. See the structure below as an example:
```javascript
class Books {
  constructor() {
     this.books = JSON.parse(localStorage.getItem('bookArchive')) || [];
  }

  addBook(title, author) {
  }

  removeBook(bookID) {
  }

  getBooks() {
  }
}
```